ArticleTemplates/assets/scss/pillar/_colour.scss

```diff
@@ -341,8 +341,76 @@
                 background: $p-feature-headline;
                 color: $p-inverted;
             }
+
+            &.personality-quiz {
+                .question__wrapper {
+                    border-top: 2px solid color(brightness-7);
+                }
+
+                .question__text {
+                    &::before {
+                        color:  color(brightness-7);
+                    }
+                }
+
+                .quiz-scores,
+                .quiz-results {
+                    border-color: color(brightness-7);
+                    border-top: 2px solid color(brightness-7);
+                }
+            }
+        }
+    }
+
+    @media (prefers-color-scheme: dark) {
+        .element-atom {
+            .quiz {
+                &.personality-quiz {
+                    .question__wrapper {
+                        border-top: 2px solid #707070;
+                    }
+
+                    .question__text {
+                        &::before {
+                            color: #707070;
+                        }
+                    }
+
+                    .quiz-scores,
+                    .quiz-results {
+                        border-color: #707070;
+                        border-top: 2px solid #707070;
+                    }
+                }
+            }
         }
     }
+
+    //for android
+    .body .dark-mode-on {
+        .element-atom {
```

Comment:
I think here you can create a mixin to share  the styles that are common between the ios and android. Something like 
```
@mixin apps-dark-mode-article {
        .element-atom {
            .quiz {
                &.personality-quiz {
                    .question__wrapper {
                        border-top: 2px solid #707070;
                    }

                    .question__text {
                        &::before {
                            color: #707070;
                        }
                    }

                    .quiz-scores,
                    .quiz-results {
                        border-color: #707070;
                        border-top: 2px solid #707070;
                    }
                }
            }
        }

.body .dark-mode-on {
  @include apps-dark-mode-article;
}

@media (prefers-color-scheme: dark) {
   @include apps-dark-mode-article;
}
}
```